projects/libreoffice/fonts.conf

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
+<fontconfig>
+  <dir>/out/instdir/share/fonts/truetype</dir>
```

Comment:
unfortunately these paths won't be available on the ClusterFuzz. AFAIK the only way to locate data files while running the fuzzer is relative to argv[0]. @oliverchang